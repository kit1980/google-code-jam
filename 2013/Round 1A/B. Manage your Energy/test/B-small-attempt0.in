100
5 1 3
1 2 3
5 1 3
1 3 2
5 1 3
2 1 3
5 1 3
2 3 1
5 1 3
3 1 2
5 1 3
3 2 1
2 1 3
3 1 3
3 1 5
1 2 3 2 1
3 1 5
3 2 1 2 3
5 1 1
2
5 5 10
10 10 10 10 10 10 10 10 10 10
5 1 10
10 1 10 1 10 1 10 1 10 1
5 1 10
1 10 1 10 1 10 1 10 1 10
5 1 10
1 2 3 4 5 6 7 8 9 10
5 1 10
10 9 8 7 6 5 4 3 2 1
5 1 10
1 10 2 9 3 8 4 7 5 6
5 1 10
10 1 9 2 8 3 7 4 6 5
5 5 1
9
1 1 1
9
1 1 1
9
1 1 1
4
4 2 1
9
2 2 1
2
2 1 1
1
3 3 1
2
3 2 1
4
4 3 1
1
2 2 1
10
5 5 1
7
5 1 1
9
3 2 1
6
2 1 1
3
5 4 1
3
1 1 1
8
1 1 1
2
5 1 1
8
4 1 1
2
1 1 1
3
1 1 1
10
1 1 1
5
4 2 1
6
5 4 1
5
2 1 1
4
3 3 1
2
3 2 1
8
5 4 1
5
2 1 1
7
3 2 1
9
3 1 1
2
3 2 1
3
4 1 1
7
5 3 1
7
5 2 1
5
4 3 1
10
3 1 1
5
5 2 1
10
4 4 1
6
3 1 1
6
3 1 1
10
1 1 1
4
1 1 1
9
3 3 1
9
3 3 1
4
1 1 1
4
1 1 1
10
5 4 1
7
2 1 1
1
4 1 1
1
5 3 1
5
5 2 1
5
1 1 1
8
1 1 1
10
5 1 1
8
1 1 1
9
3 1 1
4
4 1 1
7
4 3 1
4
3 2 1
8
2 2 1
1
3 2 1
1
3 2 1
5
2 1 1
7
1 1 1
7
2 2 1
10
4 3 1
6
5 2 1
8
4 3 1
9
1 1 1
3
4 3 1
6
4 5 1
3
5 5 1
10
5 1 2
10 7
5 2 2
10 8
5 3 2
10 10
5 4 3
4 6 10
5 3 4
9 10 10 9
5 3 5
6 5 10 6 7
5 2 6
10 10 9 10 1 8
5 2 7
7 10 5 7 7 10 6
5 1 10
5 6 10 7 10 10 8 9 7 3
