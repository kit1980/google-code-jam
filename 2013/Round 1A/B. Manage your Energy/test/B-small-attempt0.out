Case #1: 18
Case #2: 18
Case #3: 19
Case #4: 18
Case #5: 19
Case #6: 18
Case #7: 12
Case #8: 15
Case #9: 20
Case #10: 10
Case #11: 500
Case #12: 131
Case #13: 131
Case #14: 95
Case #15: 95
Case #16: 111
Case #17: 115
Case #18: 45
Case #19: 9
Case #20: 9
Case #21: 4
Case #22: 36
Case #23: 4
Case #24: 2
Case #25: 6
Case #26: 12
Case #27: 4
Case #28: 20
Case #29: 35
Case #30: 45
Case #31: 18
Case #32: 6
Case #33: 15
Case #34: 8
Case #35: 2
Case #36: 40
Case #37: 8
Case #38: 3
Case #39: 10
Case #40: 5
Case #41: 24
Case #42: 25
Case #43: 8
Case #44: 6
Case #45: 24
Case #46: 25
Case #47: 14
Case #48: 27
Case #49: 6
Case #50: 9
Case #51: 28
Case #52: 35
Case #53: 25
Case #54: 40
Case #55: 15
Case #56: 50
Case #57: 24
Case #58: 18
Case #59: 30
Case #60: 4
Case #61: 9
Case #62: 27
Case #63: 12
Case #64: 4
Case #65: 10
Case #66: 35
Case #67: 2
Case #68: 4
Case #69: 25
Case #70: 25
Case #71: 8
Case #72: 10
Case #73: 40
Case #74: 9
Case #75: 12
Case #76: 28
Case #77: 16
Case #78: 24
Case #79: 2
Case #80: 3
Case #81: 15
Case #82: 14
Case #83: 7
Case #84: 20
Case #85: 24
Case #86: 40
Case #87: 36
Case #88: 3
Case #89: 24
Case #90: 12
Case #91: 50
Case #92: 57
Case #93: 66
Case #94: 80
Case #95: 90
Case #96: 134
Case #97: 126
Case #98: 142
Case #99: 147
Case #100: 119
